src/components/simulator/LogEvent.tsx

```diff
@@ -72,6 +72,8 @@ export interface EventProps {
   classifier?: { uuid: string; name: string };
   ticketer?: { uuid: string; name: string };
   ticket?: { topic: Topic; body: string };
+  hint?: Hint;
+  timeout_seconds?: number;
```

Comment:
it'd be pretty neat if the simulator simulated timeouts and you could see a timer counting down to when you get moved on

Reply:
We talked about this in the past, but giving the user the option to trigger no-response. Could do both.